### DISCUSSION TITLE
Removing bee nests?

### DISCUSSION THREAD

**Discussion by Kadinumoto** [EwyBoy/OreTweaker#63] [Help Wanted!] [answered]
Is it possible to use ore tweaker to remove minecraft:bee_nest?

**Comment by EwyBoy** [comment, marked as answer]
No, you would need a custom mod to do that.
Ore Tweaker only tweaks ore spawn.

**Reply by Kadinumoto** [reply]
Thank you for answering! Really enjoying changing the ore generation and I saw you could exclude blocks/ores so I was wondering about the bee nests, sorry if it was a silly question!

**Reply by EwyBoy** [reply]
Thank you!
Ore Tweaker does not go block by block when disabling the ores.
Minecraft uses something called features generate things in the world. Ores are registered as `OreFeatures` and then places into the world as 1 feature and not on a block by block basis.
Ore Tweaker is listening for these features and disables them then replace it by registering its own features that can be customized on launch.
